src/SignalR/common/Protocols.MessagePack/src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack.csproj

```diff
@@ -20,4 +20,8 @@
     <Reference Include="MessagePackAnalyzer" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <SupportedPlatform Include="browser" />
```

Comment:
This one is iffy since the 3rd party package doesn't claim to be supported, same with Newtonsoft

Reply:
Adding this doesn't add any additional claims to the assembly. All it does is tell the analyzer to complain if it's using unsupported APIs for this additional platform. Right now, it tells us if we're using any unsupported APIs in our own code which is somewhat useful.

Reply:
> Adding this doesn't add any additional claims to the assembly

Ah, I thought it did :)